remove unsupported feature from public document (#1497)
vdaas/vald: README.md

    - Horizontal scalable on memory and CPU for your demand.

- Auto Backup for Index data

>     - Vald has a feature to store the backup of the each Vald Agent index data using Object Storage or Persistent Volume.

Reviewer: I think to simplify the sentences, we don't need to specify each Vald Agent.
```suggestion
    - Vald supports to backup Vald Agent index data using Object Storage or Persistent Volume.
```
